packages/electron/package.json

```diff
@@ -17,25 +17,26 @@
   "scripts": {
     "docs": "typedoc",
     "prebuild": "rimraf lib",
-    "build": "tsc",
+    "build": "yarn && ./node_modules/.bin/tsc",
     "test": "jest"
   },
   "dependencies": {
-    "async-retry": "^1.2.1",
+    "async-retry": "^1.2.3",
     "axios": "^0.18.0",
     "checksum": "^0.1.1",
-    "command-exists": "^1.2.6",
-    "debug": "^4.1.0",
-    "electron-dl": "^1.11.0",
+    "command-exists": "^1.2.8",
+    "debug": "^4.1.1",
+    "electron-dl": "^1.12.0",
     "promise-any": "^0.2.0",
-    "sntp": "^3.0.1"
+    "sntp": "^3.0.2"
   },
   "devDependencies": {
     "@types/async-retry": "^1.2.1",
     "@types/checksum": "^0.1.30",
-    "electron": "^2.0.2"
+    "electron": "^3.1.0",
```

Comment:
I want this package to be available for all electron versions. The only weak link is `electron-dl`, which I am not 100% sure is compatible with Electron 3-4, but let's assume yes until proven wrong (and do some testing on Fether). All other packages are only nodejs.

So replace with "^2.0.2 | ^3.1.0 | ^4.0.1"

Reply:
If I also update Fether's fether-electron package's package.json with:
```
"devDependencies": {
    ...
    "electron": "^2.0.2 | ^3.1.0 | ^4.0.1",
```

Then when I run `yarn; yarn start` it prompts me to choose the version, but it shows versions prior as well like 1.x.x
```
? Please choose a version of "electron" from this list: (Use arrow keys)
❯ 4.0.1 
  4.0.0 
  4.0.0-nightly.20181010 
...
  4.0.0-beta.1 
  3.1.0 
  3.1.0-beta.5 
...
  3.0.4 
(Move up and down to reveal more choices)
```

Reply:
In js-libs, we put "^2.0.2 | ^3.1.0 | ^4.0.1", which means "this library supports these versions of Electron."

In Fether, we choose one specific version of Electron (4.0.1). So in fether it should only be "^4.0.1"